packages/neon-core/src/sc/util.ts

```diff
@@ -16,7 +16,7 @@ export function isSignatureContract(input: HexString): boolean {
     script[0] != OpCode.PUSHDATA1 ||
     script[1] != PUBLIC_KEY_LENGTH ||
     script[35] != OpCode.SYSCALL ||
-    script.readUInt32LE(36) != 2859889780
+    script.readUInt32LE(36) != 666101590
```

Comment:
Can we take this opportunity to derive this magic number from the InteropServiceCode reference so it is clearer?